src/locales/swedish.js

```diff
@@ -0,0 +1,386 @@
+    NOT_SAME_VC: "Du måste vara i samma röstchatt som botten",
+    ALREADY_IN_VC: "Jag är redan i den röstchatten",
+  },
+  ECONOMY: {
+    MONEY: "Pengar",
+    BANK: "Bank",
+    BALANCE: "Balans",
+    DAILY_ERROR: "Du har redan samlat in ditt dagliga",
+    WEEKLY_ERROR: "Du har redan samlat in din veckovisa",
+    DAILY_SUCCESS: "Du samlade ditt dagliga **{amount}** mynt",
+    WEEKLY_SUCCESS: "Du har samlat in din veckovis **{amount}** mynt",
+    STORE_EMPTY:
+      "Butiken för denna server är tom! Be en moderator lägga till artiklar i butiken med `{prefix}store add <Artikel>`",
+      PROVIDE_ITEM_TO_BUY: "Vänligen ange en artikel att köpa",
+    NOT_ENOUGH_MONEY: "Du har inte så mycket pengar",
+    BUY_SUCCESS: "Framgångsrikt köpt **{item}** betalas **{price}**",
+    NOT_FOUND_STORE:
+      "**{query}** hittades inte i butiken, använd `{prefix}store` för att se alla artiklar i butiken",
+    ALREADY_OWN_ITEM: "Du äger redan det här föremålet",
+    PROVIDE_VALID_AMOUNT: "Ange ett giltigt numeriskt belopp",
+    DEPOSITED_ALL: "Har deponerat alla dina pengar!",
+    DEPOSITED_AMOUNT: "Har deponerats framgångsrikt **{amount} mynt**",
+    WITHDRAW_ALL: "Lyckades tillbaka alla dina pengar!",
+    WITHDRAW_AMOUNT: "Sframgångsrikt återkallad **{amount} mynt**",
+    PROFILE: "Profil",
+    INV_EMPTY: "Användarens inventering är tom",
+    INVENTORY: "Lager",
+    INV_ITEMS: "Lagerartiklar",
+    VIEW_INVENTORY:
+      "Använda `{prefix}inventory <user>` för att se deras lagervaror.",
+    MONEY_LEADERBOARD: "Topplista för pengar",
+    TOTAL_BALANCE: "Total balans",
+    BOTH_COUNTED: "Bank och pengar räknades båda",
+    DICE_LANDED: "Du landade på: {roll}",
+    DICE_WON: "grattis! Du vann ett pris på **{price}mynt**",
+    DICE_LOST: "Du måste landa en **6** för att få ett pris på **{price}mynt**",
+    RECENTLY_WORKED: "Du har redan arbetat nyligen, {time} återstående",
+    WORKED: "{member} arbetat som en **{job_name}** och tjänat **{amount}**!",
+    CANNOT_PAY_SELF: "Du kan inte betala själv",
+    PAY_SUCCESS: "Framgångsrikt gav **{member}** **{amount}mynt**",
+    CANNOT_ROB_SELF: "Du kan inte råna dig själv!",
+    BETWEEN_1_1000: "Beloppet måste vara mellan 1 och 1000",
+    MEMBER_NO_MONEY:
+      "Användaren har inga pengar, därför kan du inte råna den här användaren.",
+    ROB_SUCCESS: "Rånat framgångsrikt **{amount}mynt** från **{member}**",
+    STORE: "butiken",
+    MANAGE_STORE_PERMS:
+      "Du har inte rätt behörighet att hantera butiken! (Hantera server)",
+    PROVIDE_VALID_ITEM: "Vänligen ange ett giltigt objekt till add/remove!",
+    PRICE: "Price",
+    ALREADY_EXISTS: "**{item}** finns redan i butiken!",
+    PROVIDE_PRICE: "Ange ett pris för artikeln!",
+    MUST_BE_NUMBER: "Priset måste vara ett nummer!",
+    ADDED_TO_STORE: "{item} lades till i affären!",
+    NOT_IN_STORE: "**{item}** finns inte i butiken!",
+    REMOVED_FROM_STORE: "{item} togs bort från affären!",
```

Comment:
2 missing here too:

`WON_SLOTS: "You won and got {amount} coins",`
`LOST_SLOTS: "You lost!",`